src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingService.kt

```diff
@@ -379,6 +382,59 @@ class SentencingService(
     }
   }
 
+  private fun refreshCourtOrder(
+    courtEvent: CourtEvent,
+    offenderNo: String,
+  ) {
+    courtEvent.courtEventCharges.find { charge -> charge.resultCode1?.resultRequiresACourtOrder() ?: false }?.let {
```

Comment:
You could probably make this more readable by using `courtEvent.courtEventCharges.any {} `